PowerEditor/src/langs.model.xml

```diff
@@ -178,8 +178,8 @@
         </Language>
-        <Language name="ini" ext="ini inf url wer" commentLine=";"/>
-        <Language name="inno" ext="iss" commentLine=";" commentStart="" commentEnd="">
+        <Language name="ini" ext="cfg conf hhp ini inf url vmoptions wer" commentLine=";"/>

```

Comment:
cfg/conf added as ini file? This is plain wrong tbh, as these extensions are used by many applications with many different formats.